libs/@hashintel/styled-system/package.json

```
@@ -0,0 +1,52 @@
+{
+  "name": "@hashintel/styled-system",
+  "version": "0.0.1",
+  "description": "HASH Panda Styled System",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/hashintel/hash.git",
+    "directory": "libs/@hashintel/ui-preset"
+  },
+  "license": "(MIT OR Apache-2.0)",
+  "sideEffects": false,
+  "type": "module",
+  "scripts": {
+    "prepare": "panda codegen --clean",
+    "dev": "panda codegen --watch",
+    "build": "panda codegen"
+  },
+  "exports": {
+    "./css": {
+      "types": "./dist/css/index.d.ts",
+      "require": "./dist/css/index.mjs",
+      "import": "./dist/css/index.mjs"
+    },
+    "./tokens": {
+      "types": "./dist/tokens/index.d.ts",
+      "require": "./dist/tokens/index.mjs",
+      "import": "./dist/tokens/index.mjs"
+    },
+    "./types": {
+      "types": "./dist/types/index.d.ts",
+      "require": "./dist/types/index.mjs",
+      "import": "./dist/types/index.mjs"
+    },
+    "./patterns": {
+      "types": "./dist/patterns/index.d.ts",
+      "require": "./dist/patterns/index.mjs",
+      "import": "./dist/patterns/index.mjs"
+    },
+    "./jsx": {
+      "types": "./dist/jsx/index.d.ts",
+      "require": "./dist/jsx/index.mjs",
+      "import": "./dist/jsx/index.mjs"
+    },
+    "./styles.css": "./dist/styles.css"
+  },
+  "dependencies": {
+    "@hashintel/ui-preset": "0.0.0-private"
```

Needs to be published if mentioned here
```suggestion
    "@hashintel/ui-preset": "0.0.1"
```